addons/web/static/src/views/fields/radio/radio_selection_field_with_filter.js

```diff
@@ -7,11 +7,11 @@ import { registry } from "@web/core/registry";
 export class RadioSelectionFieldWithFilter extends RadioField {
     static props = {
         ...RadioField.props,
-        allowed_selection: { type: Array },
+        allowed_selection: { type: Set },
     };
 
     get items() {
-        return this.props.allowed_selection;
+        return super.items.filter((pair) => this.props.allowed_selection.has(pair[0]));
```

Comment:
Can't we write:

`super.items.filter(([value, label]) => this.props.allowed_selection.has(value));`

A bit more elegant.

Reply:
oh that's how you do it!